packages/ag-charts-community/src/chart/series/cartesian/bubbleSeries.ts

```diff
@@ -708,6 +702,20 @@ export class BubbleSeries extends CartesianSeries<
         } satisfies ResultRules;
     }
 
+    private makeLabelFormatterParams(): AgBubbleSeriesLabelFormatterParams {
+        const { xKey, xName, yKey, yName, sizeKey, sizeName, labelKey, labelName } = this.properties;
+        return {
+            xKey,
+            xName,
+            yKey,
+            yName,
+            sizeKey,
+            sizeName,
+            labelKey,
+            labelName,
+        } satisfies RequireOptional<AgBubbleSeriesLabelFormatterParams>;
```

Comment:
Could you make the return type of the function `RequireOptional<AgBubbleSeriesLabelFormatterParams>`?